book/release-service.md

```diff
@@ -39,8 +28,95 @@ The diagram below shows the flow of custom resources and the orchestration of pi
 
 ![](../diagrams/release-service/hacbs-release-service-data-flow.jpg)
 
+## Terminology
+
+**Snapshot** -  The custom resource that contains the list of all Components of an Application with their Component Image digests. Once created, the list of Components with their images is immutable. The Integration service updates the status of the resource to reflect the testing outcome.
+**Release Pipeline** -  An instance of a Pipeline that is orchestrated by the Release Service.
+**Bundle** - A tekton bundle (image) that contains Tekton resources
+**Development Workspace** - workspace shared by 1 or more team members when code is built and tested.
+**Managed Workspace** - workspace that is managed by an SRE or Release Engineering team that is controlled. This is where Release pipeline execute.
```

Comment:
The first 3 start with a capital after the hyphen. I think "A controlled workspace that is managed by an SRE or Release Engineering team" reads better. Kind of sounds like the Release Engineering team is controlled as written. Also, `s/execute/executes/`

Reply:
the p in Release Pipeline is capitalized seemingly everywhere else. Should probably be consistent

Reply:
fixed